### DISCUSSION TITLE
Different log formats for two json payloads

### DISCUSSION THREAD

**Discussion by woolamkang** [emqx/emqx#8120] [Q&A]
I use emqx 3.2.7
Some log of published payload is following.

[Protocol] RECV PUBLISH(Q1, R0, D0, Topic=xxx/command/yyy, PacketId=10262, Payload=<<"{\"affect\":\"admin\"}">>) 

But some log of published payload is following.

[Protocol] RECV PUBLISH(Q1, R0, D0, Topic=xxx/result/yyy, PacketId=130, Payload=<<123,34,
                                                                              115,99,
                                                                              104,101,
                                                                              100,117,
                                                                              108,101,
                                                                              34,58,
...

Both of two payloads are JSON messages.
Why these are written as different format?
Can I handle this to use readable format?

**Comment by zmstone** [comment]
Hi @woolamkang 3.x is out of maintenance window. 
Please consider upgrading to 4.4
